collections/_get-started/customise-with-sass.html

```diff
@@ -1,33 +1,128 @@
-                    <img src="/assets/img/figmalogo.png" class="img-fluid" alt="Figma logo">
-                  </div>
-                  <div class="col-md-9">
-                    <div class="card-body">
-                      <h5 class="card-title">Figma UI Kit</h5>
-                      <a class="card-text"><small class="text-muted">Last updated 3 mins ago</small></a>
-                    </div>
-                  </div>
+            <p>The Sass source files for sgds can be found in the folder<br><code>node_modules/@govtechsg/sgds/sass</code></p>
+            {% highlight bash %}{{sass-folder-structure}}{% endhighlight %}
+            
+              <div class="sgds toast show w-100" role="alert" aria-live="assertive" aria-atomic="true">
+                
+                <div class="toast-body pt-3">
+                  Whenever possible, avoid modifying the core sass files in <code>bootstrap</code> and <code>sgds-theme</code>
                 </div>
               </div>
+            
         </div>
     </section>
-    <section name="section-dss">
-        <h2 class="title fw-bold" id="anatomy">Digital Service Standards</h2>
-        <p>To complement the visual style, we highly recommend going through the Digital Service Standards (DSS), to ensure that you adhere to the policies/clauses stated. Even though our components are built and designed with the clauses in place where applicable, there are instances where they can’t be designed or coded.</p>
+    <section name="section-importing">
+      <div class="section-sub">
+        <h2 class="title fw-bold" id="anatomy">Importing</h2>
+        <p>In your <code>custom.scss</code>, you can choose to include all <code>sgds-theme</code> files or pick the components you need. However do note that are some requirements and dependencies across the components.</p>
+      </div>
+      <div class="section-sub">
+        <h3 class="title fw-bold" id="anatomy">Include all parts</h3>
+        {% highlight css %}{{include-all-parts}}{% endhighlight %}
+      </div>
+      <div class="section-sub">
+        <h3 class="title fw-bold" id="anatomy">Include parts of SGDS</h3>
+        {% highlight css %}{{include-parts}}{% endhighlight %}
+
+        <p>If you are using Webpack, u can omit the <code>node_modules</code> directory</p>
+        {% highlight css %}{{webpack-sass}}{% endhighlight %}
+      </div>
+    </section>
+    <section name="section-design-tokens">
+      <h2 class="title fw-bold" id="anatomy">Variables</h2>
+      <p>
+        You can find the complete list of <code>sgds-theme</code> variables inside <code>sass/sgds-theme/_sgds-variables.scss</code> or go to our github page <a href="{{site.sgds-variables-github}}" target="_blank" rel="noopener noreferrer">here</a>
+      </p>
+      
+      
+      <div class="section-sub">
+        <p>Here's an example that changes the background-color and color for the <body> when importing and compiling <code>sgds-theme</code> via npm:</p>
```

Comment:
color for the ...<body>.. not seen in browser

Reply:
content updated in commit [0388edf](https://github.com/GovTechSG/sgds/pull/369/commits/0388edfb0e24d0c636661b6bdbf253f8f94ff4e1)